tests/manager/manager-ctor-ssl-deprecated-001.phpt

```diff
@@ -0,0 +1,63 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): Test deprecated options
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_ssl(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$opts = array(
+    "ssl" => array(
+        "verify_peer" => false,
+        "verify_peer_name" => false,
+        "allow_self_signed" => true,
+    ),
+);
+
+$deprecatedDriverOptions = [
+    ['allow_invalid_hostname' => true],
+    ['weak_cert_validation' => true],
+    ['allow_self_signed' => true],
+    ['pem_file' => 'foo'],
+    ['local_cert' => 'foo'],
+    ['pem_pwd' => 'foo'],
+    ['passphrase' => 'foo'],
+    ['ca_file' => 'foo'],
+    ['cafile' => 'foo'],
+    ['context' => stream_context_create($opts)],
```

Comment:
Ah, I now see how `$deprecatedDriverOptions` interacts with the `foreach` below. In that case, I think you'll need new test cases for each of the deprecated context options. In that case, the earlier `$opts` definition can probably be removed entirely as it will make more sense to define the five distinct variations inline in the calls to `stream_context_create()`.